### PR TITLE
add hck14: NWB data mining

### PR DESCRIPTION
**Pull request by bendichter**

### CODE REVIEW

File: HCK14_2022_Seattle/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  * [Resources](#resources)
+  * [What to bring?](#what-to-bring)
+  * [Objective](#objective)
+  * [Projects](#projects)
+     * [Creating a New Project](projects/README.md)
+  * [Disclaimer](#disclaimer)
+  
+## Dates and Location
+
+- **Dates:** September 12-15, 2022
```
Comment (rly) [inline HCK14_2022_Seattle/README.md:17]: Should this be September 12-14?

File: HCK14_2022_Seattle/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+## Resources
+
+### Talks
+
+Recordings of talks will be made available on the [NWB Youtube channel](https://www.youtube.com/channel/UCfD_mU-EFz135a9TpNFJP5A).
+
+## What to bring?
+
+* Create the outline for your project at the hackathon. For further details and instructions on how to create a project, see [here](projects/README.md)
+* Bring your laptop with appropriate software installed. For instructions on how to install PyNWB, see [http://pynwb.readthedocs.io/en/latest/getting_started.html#installation](http://pynwb.readthedocs.io/en/latest/getting_started.html#installation)
```
Comment (rly) [inline HCK14_2022_Seattle/README.md:53]: Any reason why we call out PyNWB and not MatNWB?
```suggestion
* Bring your laptop with appropriate software installed. For instructions on how to install PyNWB, see [the PyNWB documentation](http://pynwb.readthedocs.io/en/latest/getting_started.html#installation). For instructions on how to install MatNWB, see [the MatNWB documentation](https://github.com/NeurodataWithoutBorders/matnwb/blob/master/README.md)
```

File: README.md [hunk 3]
```diff
@@ -4,6 +4,8 @@
 
 ### Upcoming Events
 - [Hackathon 13: July 2022](HCK13_2022_Janelia/README.md){:target="_top"} - The 13th Hackathon: User Days will be held at the HHMI Janelia Research Campus in Ashburn, Virginia, USA on **July 25 - 28, 2022**.
+- [Hackathon 14: September 2022](HCK14_2022_Seattle/README.md){:target="_top"} - The 14th Hackathon: NWB 
+  Neurophysiology Data Mining at the Allen Institute in Seattle, Washington, USA on **September 12-15, 2022**.
```
Comment (rly) [inline README.md:8]:
```suggestion
  Neurophysiology Data Mining at the Allen Institute in Seattle, Washington, USA on **September 12-14, 2022**.
```

File: HCK14_2022_Seattle/README.md (hunk 1 above)
Comment (bendichter) [reply HCK14_2022_Seattle/README.md:17]:
```suggestion
- **Dates:** September 12-14, 2022
```

**Pull request merged**